src/components/mozilla/footer.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Footer from '../footer.jsx';
+import Footer from '../footer.js';
```

Comment:
These file names are changed, because technically this is being run as dist, after it's compiled, so technically it's running js and not jsx.